core/sdk/src/api/device/device_user_grants_data.rs

```diff
@@ -45,7 +45,7 @@ pub struct GrantRequirements {
     pub steps: Vec<GrantStep>,
 }
 
-#[derive(Eq, Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[derive(Eq, Serialize, Deserialize, Debug, Clone, PartialEq, Copy)]
```

Comment:
Why do we need Copy here when we have a clone

Reply:
Right, we don't need to add Copy here.